modules/mono/build_scripts/mono_configure.py

```diff
@@ -128,27 +128,23 @@ def get_runtime_path():
 
 
 def determine_runtime_identifier(env):
+    # The keys are Godot's names, the values are the Microsoft's names.
+    # List: https://docs.microsoft.com/en-us/dotnet/core/rid-catalog
     names_map = {
         "windows": "win",
         "macos": "osx",
         "linuxbsd": "linux",
         "server": "linux",  # FIXME: Is server linux only, or also macos?
```

Comment:
I thought the server platform was removed.

Reply:
It is removed from the engine, and will be removed here when https://github.com/godotengine/godot/pull/64846 is merged.